mmv1/products/dialogflowcx/api.yaml

```diff
@@ -474,7 +474,7 @@ objects:
     async: !ruby/object:Api::OpAsync
       operation: !ruby/object:Api::OpAsync::Operation
         path: 'name'
-        base_url: '{{op_id}}'
+        full_url: 'https://dialogflow.googleapis.com/v3/{{op_id}}'
```

Comment:
Are we missing the information to call https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.operations? It seems unusual we'd use the regional endpoint to provision & global endpoint to get the operation.

Reply:
Added {{location}} in the next PR